addons/http_routing/models/ir_http.py

```diff
@@ -575,10 +575,9 @@ def _pre_dispatch(cls, rule, args):
 
     @classmethod
     def _frontend_pre_dispatch(cls):
-        request.lang = request.lang.with_env(request.env)
```

Comment:
Do you know why this env change was needed?

Reply:
I guess it was used to promise all context and the user of `request.lang.env` was the same as `request.env`

Reply:
It was really added on purpose with https://github.com/odoo/odoo/commit/584a172274caefb23e5d91b609fc893160535416 (just this line here)

Maybe not needed anymore but worth triple checking? Didn't read the mentioned commit diff
But if you checked and seems fine, fine to me